public/js/wp-agora-io-public.js

```diff
@@ -748,7 +818,9 @@ jQuery(document).ready(function(){
   const THRESHOLD_AUDIO_LEVEL = 0.1;
   setInterval(() => {
 
-    window.allStreams = Object.fromEntries(Object.entries(window.allStreams).filter(([_, v]) => v != null));
+    if(typeof window.allStreams!='undefined'){
+      window.allStreams = Object.fromEntries(Object.entries(window.allStreams).filter(([_, v]) => v != null));
```

Comment:
what is the point of this?

Reply:
> what is the point of this?

This was of that array that we have already removed.(to maintain local and remote streams). Kindly check it.